### PR TITLE
cleaned up the game a little bit, clear screen with each scene, if pu…

### PR DESCRIPTION
**Pull request by kaashis**
…zzle is attempted, it's gone from the room

### CODE REVIEW

File: src/main/java/com/zombiecastlerush/util/Parser.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 public class Parser {
 
-    final static List<String> ALLOWED_ACTIONS = Arrays.asList("go", "look", "pick-up", "drop", "attempt","display");
+    final static List<String> ALLOWED_ACTIONS = Arrays.asList("go", "look", "pick-up", "drop", "attempt","display","quit");
```
Comment (chaynsaw) [inline src/main/java/com/zombiecastlerush/util/Parser.java:9]: It's minor, but please put a space between after the comma and before your next argument, lol. Like "pick-up", then space, then "drop", and not how it's done later with display and quit.

Alternatively just use cmd-opt-L in intellij.

File: src/main/java/com/zombiecastlerush/util/Prompter.java [hunk 2]
```diff
@@ -23,61 +23,59 @@ public static String getUserInput(String displayMessage) {
     }
 
     static void advanceGame(Player player) throws JsonProcessingException {
+        displayCurrentScene(player);
```
Comment (chaynsaw) [inline src/main/java/com/zombiecastlerush/util/Prompter.java:26]: I like the naming here. Good job.

**Review by chaynsaw — APPROVED** [review]
I approve.

**Review by gtulkn12 — APPROVED** [review]
looks good ready to merge

**Pull request merged**